web/src/components/storage/VolumeForm.jsx

```diff
@@ -19,19 +19,17 @@
  * find current contact information at www.suse.com.
  */
 
-import React, { useReducer } from "react";
-
+import React, { useReducer, useState } from "react";
 import {
-  InputGroup, InputGroupItem,
-  Form, FormGroup, FormSelect, FormSelectOption,
-  Radio,
-  TextInput,
+  InputGroup, InputGroupItem, Form, FormGroup, FormSelect, FormSelectOption, MenuToggle,
+  Popover, Radio, Select, SelectOption, SelectList
 } from "@patternfly/react-core";
 import { sprintf } from "sprintf-js";
 
 import { _, N_ } from "~/i18n";
 import { FormValidationError, If, NumericTextInput } from '~/components/core';
 import { DEFAULT_SIZE_UNIT, SIZE_METHODS, SIZE_UNITS, parseToBytes, splitSize } from '~/components/storage/utils';
+import { Icon } from "../layout";
```

Comment:
It should be consistent with the other imports:

```suggestion
import { Icon } from "~/components/layout";
```